JitsiConference.ts

```diff
@@ -1737,9 +1865,11 @@ export default class JitsiConference {
      * Mutes a participant.
      * @param {string} id The id of the participant to mute.
      */
-    muteParticipant(id, mediaType) {
-        if (!mediaType) {
-            logger.error('muteParticipant: mediaType is required');
+    muteParticipant(id: string, mediaType: MediaType): void {
+        const muteMediaType = mediaType ? mediaType : MediaType.AUDIO;
```

Comment:
If the parameeter is optional, annotate it as such and give it a default value, rather than. doing it like this.